### ISSUE TITLE
Get a increasing val loss in training LSTM

### ISSUE THREAD

**Issue by liupeiye** [tflearn/tflearn#441]
I want to train a lstm network, but the val's loss is increasing or shaking, am i  wrong about my network?
![image](https://cloud.githubusercontent.com/assets/13778786/19998918/6916e236-a2aa-11e6-80ce-e49e4b1b0aca.png)

![image](https://cloud.githubusercontent.com/assets/13778786/19998985/c7790890-a2aa-11e6-8c84-a44fdda3848a.png)

**Comment by kirit93** [comment]
I've had similar problems in other LSTM implementations that I've used. I find that decreasing the learning rate helps in such cases. Try reducing your learning rate and decay and you might see better loss values.
